content/en/blog/_posts/2025-04-23-Kubernetes-v1-33-Release/index.md

```diff
@@ -0,0 +1,453 @@
+
+This work was done as part of [KEP-2902: Add CPUManager policy option to distribute CPUs across NUMA nodes instead of packing them](https://kep.k8s.io/2902) led by SIG Node.
+
+### Zero-second sleeps for container PreStop hooks
+
+Kubernetes 1.29 introduced a Sleep action for the `preStop` lifecycle hook in Pods, allowing containers to pause for a specified duration before termination.
+This provides a straightforward method to delay container shutdown, facilitating tasks such as connection draining or cleanup operations.
+
+The Sleep action in a `preStop` hook can now accept a zero-second duration as a beta feature. This allows defining a no-op `preStop` hook, which is useful when a `preStop` hook is required but no delay is desired.
+
+This work was done as part of [KEP-3960: Introducing Sleep Action for PreStop Hook](https://kep.k8s.io/3960) and [KEP-4818: Allow zero value for Sleep Action of PreStop Hook](https://kep.k8s.io/4818) led by SIG Node.
+
+### Declarative validation of Kubernetes-native types
+
+Behind the scenes, the internals of Kubernetes are starting to use a new mechanism
+for validating objects and changes to objects.
+Kubernetes v1.33 introduces `validation-gen`, an internal tool that Kubernetes
+contributors use to generate
+declarative validation rules. The overall goal is to improve the robustness and maintainability of API validations
+by enabling developers to specify validation constraints declaratively, reducing manual coding errors
+and ensuring consistency across the codebase.
+
+This work was done as part of [KEP-5073: Declarative Validation Of Kubernetes Native Types With validation-gen](https://kep.k8s.io/5073) led by SIG API Machinery.
+
+## New features in Alpha
+
+*This is a selection of some of the improvements that are now alpha following the v1.33 release.*
+
+### Configurable tolerance for HorizontalPodAutoscalers
+
+This feature introduces configurable tolerance for HorizontalPodAutoscalers, which dampens scaling reactions to small metric variations.
+
+This work was done as a part of [KEP-4951: Configurable tolerance for Horizontal Pod Autoscalers](https://kep.k8s.io/4951) led by SIG Autoscaling.
+
+### Configurable container restart delay
+
+Introduced as alpha1 in v1.32, this feature provides a set of kubelet-level configurations to fine-tune how CrashLoopBackOff is handled.
+
+This work was done as part of [KEP-4603: Tune CrashLoopBackOff](https://kep.k8s.io/4603) led by SIG Node.
+
+### Custom container stop signals
+
+Before Kubernetes v1.33, stop signals could only be set in container image definitions (for example, via the `StopSignal` configuration field in the image metatadata).
```

Comment:
```suggestion
Before Kubernetes v1.33, stop signals could only be set in container image definitions (for example, via the `StopSignal` configuration field in the image metadata).
```

Reply:
I even had spell check on while making the final edits... I don't know how I didn't catch this 😅